bpo-34429: Noted TemporaryFile behavior on non-Posix/non-Cygwin systems (#31547)
python/cpython: Doc/library/tempfile.rst

   The :py:data:`os.O_TMPFILE` flag is used if it is available and works
   (Linux-specific, requires Linux kernel 3.11 or later).

>    On platforms that are neither Posix nor Cygwin, TemporaryFile behaves exactly
>    like NamedTemporaryFile.

Reviewer: ```suggestion
   On platforms that are neither Posix nor Cygwin, TemporaryFile is an alias
   for NamedTemporaryFile.
```
